felix/rules/static.go

```diff
@@ -1144,40 +1144,53 @@ func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8,
 		Rules: rawRules,
 	}
 
-	bpfUntrackedFlowChain := &Chain{
-		Name:  ChainRawUntrackedFlows,
-		Rules: []Rule{},
-	}
+	// BPF Untracked Flow Rules:
+	var bpfUntrackedFlowRules []Rule
+	var bpfUntrackedFlowChain *Chain
 
 	if bypassHostConntrack {
-		bpfUntrackedFlowChain = &Chain{
-			Name: ChainRawUntrackedFlows,
-			Rules: []Rule{
-				Rule{
-					Match:   Match().MarkMatchesWithMask(tcdefs.MarkSeenSkipFIB, tcdefs.MarkSeenSkipFIB),
-					Action:  ReturnAction{},
-					Comment: []string{"MarkSeenSkipFIB Mark"},
-				},
-				Rule{
-					Match:   Match().MarkMatchesWithMask(tcdefs.MarkSeenFallThrough, tcdefs.MarkSeenFallThroughMask),
-					Action:  ReturnAction{},
-					Comment: []string{"MarkSeenFallThrough Mark"},
-				},
-				Rule{
-					Match:   Match().MarkMatchesWithMask(tcdefs.MarkSeenMASQ, tcdefs.MarkSeenMASQMask),
-					Action:  ReturnAction{},
-					Comment: []string{"MarkSeenMASQ Mark"},
-				},
-				Rule{
-					Match:   Match().MarkMatchesWithMask(tcdefs.MarkSeenNATOutgoing, tcdefs.MarkSeenNATOutgoingMask),
-					Action:  ReturnAction{},
-					Comment: []string{"MarkSeenNATOutgoing Mark"},
-				},
-				Rule{
-					Action: NoTrackAction{},
-				},
-			},
+		// Iterate all BPF interfaces forced to track packets and append rule.
+		for _, interfaceName := range r.BPFForceTrackPacketsFromIfaces {
+			if len(interfaceName) > 0 {
```

Comment:
Should add a UT for the new feature

Reply:
Additional unit tests for BPF mode raw chains